explanation/application-streaming.md

```diff
@@ -43,6 +61,16 @@ When optimal ICE candidates have been selected and codec capabilities have been
 
 The streams can then be consumed to display the final content.
 
+## Streaming an application by creating a stream-enabled instance
+
+Starting from 1.22.0, Anbox Cloud offers a much simpler way to create instances that can be streamed. This is the alternative implementation depicted in the streaming stack diagram.
+
+You can request AMS to create an instance and AMS will be able to dynamically create a session for the newly launched instance by contacting the stream agent. The agent mediates communication between stream gateway and AMS and provides the necessary session information to the AMS. The AMS proceeds with successfully creating an instance that can be streamed.
+
+[note type="information" status="Note"]
```

Comment:
I wonder if that should be a warning instead, to make it more visible?

Reply:
I think keep it informative makes sense. It's not that something gets broken when streaming isn't enabled and it might be not on purpose. I see this more as a hint.

Reply:
Keeping this as a note as is. Warning is better used for breaking stuff.